cmd/get/upgrade-paths.go

```diff
@@ -0,0 +1,291 @@
+package get
```

Comment:
Missing license notice

Reply:
yeah.. I saw it in the CI :D, thanks
fixed